h_transport_materials/property_database/ss_304/ss_304.py

```diff
@@ -131,6 +132,7 @@
     source="braun_determination_1980",
 )
 
+# @FuerstT: I'm having a hard time finding the original ref for Hawkins solubility. This differs from your review. Any thoughts?
```

Comment:
@FuerstT  I'm having a hard time finding the original ref for Hawkins solubility: N.L. Hawkins, Report KAPL 863 (1953)

The diffusivity/permeability and dissociation coefficient of Braun uses it.
Braun's dissociation coeff seems to differ from your review.
Any thoughts? Do you remember how you obtained it?